spacy/ml/callbacks.py

```diff
@@ -23,6 +23,7 @@
     "update",
     "rehearse",
     "get_loss",
+    "get_student_teacher_loss",
```

Comment:
While slightly less descriptive/accurate, I wonder whether we could just call this `get_distill_loss`?

Reply:
IIRC, the reason we went for this was because we use the same loss in the model rehearsal process as well. So, it made sense not to use `distill` in the name.